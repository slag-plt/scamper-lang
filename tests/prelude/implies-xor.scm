(implies #t #f)
(implies #f #f)
(implies #t #t)
(implies #f #t)
(xor #t #f)
(xor #f #f)
(xor #t #t)
(xor #f #t)