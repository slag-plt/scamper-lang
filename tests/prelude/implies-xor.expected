#f
#t
#t
#t
#t
#f
#f
#t
